dev-client/datatypes/sites.ts

```diff
@@ -0,0 +1,17 @@
+export type Point2D = [number, number];
+
+export class DisplaySite {
+    lat: number;
+    lon: number;
+    name: string;
+
+    constructor(lat: number, lon: number, name: string) {
+        this.lat = lat;
+        this.lon = lon;
+        this.name = name;
+    }
+
+    get key() {
+        return `{this.lat},{this.lon},{this.name}`;
+    }
```

Comment:
question: what is this for?

Reply:
This is to generate a unique key for rendering in a list with React. @shrouxm Any idea how this is currently done in the frontend?

Reply:
ohh right, hmm no idk. i would think it would eventually be the object's pk if possible, but not sure what we're doing currently